### PR TITLE
[lensfun] Remove cpu features

### PR DESCRIPTION
**Pull request by JavierMatosD**
I merged https://github.com/microsoft/vcpkg/pull/34783 in error. This PR removes the CPU features.

### CODE REVIEW

File: ports/lensfun/portfile.cmake [hunk 1]
```diff
@@ -26,7 +21,6 @@ endif()
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
-        ${FEATURE_OPTIONS}
```
Comment (Osyotr) [inline ports/lensfun/portfile.cmake:29]: `-DBUILD_FOR_SSE=OFF`?

**Review by Osyotr — APPROVED** [review]

File: ports/lensfun/portfile.cmake [hunk 2]
```diff
@@ -35,6 +29,7 @@ vcpkg_cmake_configure(
         -DINSTALL_PYTHON_MODULE=ON
         -DINSTALL_HELPER_SCRIPTS=OFF
         -DBUILD_LENSTOOL=OFF
+        -DBUILD_FOR_SSE=OFF
```
Comment (Osyotr) [inline ports/lensfun/portfile.cmake:32]:
```suggestion
        -DBUILD_FOR_SSE=OFF
        -DBUILD_FOR_SSE2=OFF
```

**Review by Osyotr — APPROVED** [review]

File: ports/lensfun/portfile.cmake [hunk 3]
```diff
@@ -35,6 +29,8 @@ vcpkg_cmake_configure(
         -DINSTALL_PYTHON_MODULE=ON
         -DINSTALL_HELPER_SCRIPTS=OFF
         -DBUILD_LENSTOOL=OFF
+        -DBUILD_FOR_SSE=OFF
+        -DBUILD_FOR_SSE2=OFF
```
Comment (JackBoosY) [inline ports/lensfun/portfile.cmake:33]: Should we set this to `ON` since this port currently only support `!arm`?
cc @Neumann-A

**Pull request merged**